.releaserc.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
-    branches: ["main"],
+    branches: [
+        {name: 'v1', range: '0.x.x', channel: '0.x.x'},
```

Comment:
does this make sense? range 0?

Reply:
as we've never release a 1.0 version the current is still 0.3.5

Reply:
ok but in that case would the branch no be v1?